lib/Raven/Client.php

```diff
@@ -881,9 +1016,12 @@ public function capture($data, $stack = null, $vars = null)
             }
 
             if (!isset($data['stacktrace']) && !isset($data['exception'])) {
-                $data['stacktrace'] = array(
-                    'frames' => Stacktrace::fromBacktrace($this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__, isset($stack['line']) ? $stack['line'] : __LINE__)->getFrames(),
-                );
+                $data['stacktrace'] = [
+                    'frames' => Stacktrace::fromBacktrace(
+                        $this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__,
+                        isset($stack['line']) ? $stack['line'] : __LINE__
```

Comment:
>Maybe the good thing is to set

That would definitely work, although it seems a bit strange from a code perspective (at least in my opinion). I don't really see why this line should be splitted into multiple lines (also it makes no sense to me to move the first 3 arguments to one line and the other to another one), but if you really want to do that use the piece of code you posted to set the correct line in the stacktrace. Attached you can find some screenshots of the comparison between before and after your change

### Before
![before](https://cloud.githubusercontent.com/assets/1770485/25559499/9dc6d6d2-2d3b-11e7-87d5-f3414ab1b77c.png)

### After
![after](https://cloud.githubusercontent.com/assets/1770485/25559498/9dc52314-2d3b-11e7-9691-819b55077f48.png)

### After change with `__LINE__ - 2`
![after_line_subtraction](https://cloud.githubusercontent.com/assets/1770485/25559497/9da32052-2d3b-11e7-9bef-6e9367f41001.png)

Now, although indeed there will be a lot of cases where something similar to the last screenshot happens in user code (and consequently we can't do anything to colorize the correct lines in Sentry) I'm not sure this change is worth. But these are just my two cents